### PR TITLE
fix #70882 krunker.io

### LINKED ISSUE #70882: krunker.io
### Issue URL (Anti Adblock Script)
 [https://krunker.io/?game=MIA:g6pk6](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fkrunker.io%2F%3Fgame%3DMIA%3Ag6pk6)
### Comment
Username: @wernser412
> Cuando hago click en "claim kr" me sale una alerta, cuando hago click sin adguard activado me sale normal

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>
<details>
  <summary>Screenshot 2</summary>

  Screenshot 2
</details>
<details>
  <summary>Screenshot 3</summary>

  Screenshot 3
</details>
<details>
  <summary>Screenshot 4</summary>

  Screenshot 4
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard Browser extension v3.5.31
Browser: | Chrome
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base<br/><br/><b>Privacy:</b><br/> AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/> AdGuard Social Media<br/><br/><b>Annoyances:</b><br/> AdGuard Annoyances<br/><br/><b>Language-specific:</b><br/> AdGuard Russian, <br/>AdGuard Spanish/Portuguese

### PR DESCRIPTION
**Pull request by stanislav-atr**
#70882

Some of these rules wont affect the page instantly, but removing them will trigger antiadb script later.

### CODE REVIEW

**Comment by Alex-302** [comment]
It does not work without AdGuard on my end
<details><summary>Screenshot:</summary>

image

</details>

**Comment by stanislav-atr** [comment]
> It does not work without AdGuard on my end

Works on my side atm. It might be better to go for direct URL `krunker.io`  as it sometimes wont load correctly if coming from `krunker.io/&game=*
`

File: EnglishFilter/sections/antiadblock.txt [hunk 1]
```diff
@@ -554,6 +554,17 @@ trade2win.com#%#//scriptlet("abort-current-inline-script", "$", "!document.getEl
 @@||encodinghub.com/wp-content/uploads/*.nz/ads/
 ! https://github.com/AdguardTeam/AdguardFilters/issues/72796
 @@||toptechtalk.xyz^$generichide
+! https://github.com/AdguardTeam/AdguardFilters/issues/70882
```
Comment (Alex-302) [inline EnglishFilter/sections/antiadblock.txt:557]: Please try to block some requests by `redirect rules`(In addition to these rules, with `important`)

File: EnglishFilter/sections/antiadblock.txt [hunk 2]
```diff
@@ -554,6 +554,17 @@ trade2win.com#%#//scriptlet("abort-current-inline-script", "$", "!document.getEl
...
+@@||pagead2.googlesyndication.com/pagead/js/adsbygoogle.js$script,redirect=googlesyndication-adsbygoogle,important,domain=krunker.io
+@@||securepubads.g.doubleclick.net/tag/js/gpt.js$script,redirect=googletagservices-gpt,important,domain=krunker.io
+@@||secure.quantserve.com/quant.js$script,redirect=nooptext,important,domain=krunker.io
```
Comment (Alex-302) [inline EnglishFilter/sections/antiadblock.txt:560]: exclusions?

**Review by Alex-302 — CHANGES_REQUESTED** [review]

**Review by zloyden — APPROVED** [review]

**Pull request merged**